coord/src/module.c

```diff
@@ -2005,6 +2014,31 @@ void setHiredisAllocators(){
   hiredisSetAllocators(&ha);
 }
 
+
+void Coordinator_CleanupModule(void) {
+  MR_Destroy();
+  GlobalSearchCluser_Release();
+}
+
+void Coordinator_ShutdownEvent(RedisModuleCtx *ctx, RedisModuleEvent eid, uint64_t subevent, void *data) {
+  RedisModule_Log(ctx, "notice", "%s", "Begin releasing Coordinator resources on shutdown");
+  Coordinator_CleanupModule();
```

Comment:
`RediSearch_InitModuleInternal` is called by the coordinator.
Shouldn't `RediSearch_CleanupModule` be called before `Coordinator_CleanupModule`?